CODE_OF_CONDUCT.md

```diff
@@ -8,6 +8,9 @@ Examples of unacceptable behavior by participants include the use of sexual lang
 
 Project maintainers have the right and responsibility to remove, edit, or reject comments, commits, code, wiki edits, issues, and other contributions that are not aligned to this Code of Conduct. Project maintainers who do not follow the Code of Conduct may be removed from the project team.
 
+This code of conduct applies both within project spaces and in public spaces wh
+ere an individual explicitly associates their presence there with the project.
```

Comment:
If this is acceptable to Opal, (rather than the version adopted by the CoC project), I would suggest submitting it there as a possible minor version increment.